src/patternfly/components/Divider/examples/Divider.md

```diff
@@ -4,24 +4,47 @@ section: components
 cssPrefix: pf-c-divider
 ---
 
+import './Divider.css'
+
 ## Examples
 ```hbs title=<hr>
-{{#> divider}}{{/divider}}
+{{> divider}}
 ```
 
 ```hbs title=<li>
 <ul>
   <li>List item one</li>
-  {{#> divider divider--type="li"}}{{/divider}}
+  {{> divider divider--type="li"}}
   <li>List item two</li>
 </ul>
 ```
 
 ```hbs title=<div>
-{{#> divider divider--type="div"}}{{/divider}}
+{{> divider divider--type="div"}}
+```
+
+```hbs title=Inset-medium
+{{> divider divider--type="div" divider--modifier="pf-m-inset-md"}}
+```
+
+```hbs title=Md-inset,-no-inset-on-md,-3xl-inset-on-lg,-lg-inset-on-xl
+{{> divider divider--type="div" divider--modifier="pf-m-inset-md pf-m-inset-none-on-md pf-m-inset-3xl-on-lg pf-m-inset-lg-on-xl"}}
+```
+
+```hbs title=Vertical
+{{> divider divider--type="div" divider--modifier="pf-m-vertical pf-m-inset-md"}}
+```
+
+```hbs title=Vertical,-inset-medium
+{{> divider divider--type="div" divider--modifier="pf-m-vertical pf-m-inset-md"}}
+```
+
+```hbs title=Vertical,-md-inset,-no-inset-on-md,-lg-inset-on-lg,-sm-inset-on-xl
```

Comment:
Why does this have medium inset and no inset on medium?

Reply:
Its purpose is to illustrate removing inset.